design.md

```diff
@@ -1,13 +1,47 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+I am designing this language for those who do not speak picobot but wish to be able to provide him with instructions that are fairly natural langauge esc. 
```

Comment:
I also wanted to create a more user-friendly picobot language by basing it more heavily in english